packages/eslint-config/eslintrc.js

```diff
@@ -154,6 +159,7 @@ module.exports = {
     '@typescript-eslint/prefer-nullish-coalescing': 'error',
     '@typescript-eslint/no-extra-non-null-assertion': 'error',
     '@typescript-eslint/return-await': 'error',
+    'tsdoc/syntax': 'warn',
```

Comment:
Please use `error` to fail the build on tsdoc syntax errors. Most people ignore warnings (one of the reasons are that CI will pass and won't tell you on GitHub about the warnings), so over time we will end up with `npm lint` printing a lot of warnings.

Reply:
I'm testing with `warn` for now and will switch to `error` once we fix all issues.